catbot-runner.js

```diff
@@ -75,31 +73,34 @@ CatRunner.prototype.start = function() {
 
 CatRunner.prototype.loader = function(moduleName) {
 	// don't throw if moduleName doesn't exist.
-	try { return require(moduleName); } catch (e) { console.log(e); };
-}
+	try { return require(moduleName); } catch (e) {console.log(e); };
+};
+
+CatRunner.prototype.shouldInvokeOn = function(message) {
+	return (message.type == 'message' && message.text.match(this.regex));
```

Comment:
This is the one line in the handler that will probably be run a *lot* (possibly for every single message on the org). I would _prefer_ to keep the comparison extremely fast (comparing the first character) rather than evaluating a regex. It might not be a big deal... what do you think?